AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.scss

```diff
@@ -0,0 +1,16 @@
+@import '~govuk-frontend/govuk/base';
+@import '~govuk-frontend/govuk/all';
```

Comment:
can remove all

Reply:
tried removing - both are needed otherwise it doesnt compile

Reply:
have you looked into what components are required. by importing the size of the stylesheet is unnecessarily big for this component. it might just be the colours or a certain component you need